react/features/base/components/participants-pane-list/ListItem.tsx

```diff
@@ -194,15 +193,15 @@ const ListItem = ({
 }: Props) => {
     const styles = useStyles();
     const _isMobile = isMobileBrowser();
-    let timeoutHandler;
+    let timeoutHandler: ReturnType<typeof setTimeout>;
```

Comment:
I thought TS had TimeoutID for this?

Reply:
It does, it's just weird. In order to use TimeoutID you need to call `window.setTimeout` instead of `setTimeout`

Reply:
I've seen us do that, let's try to be consistent

Reply:
So let;'s change it to window.setTimeout then, this reads very weird.